### ISSUE TITLE
npm install does not work a second time

### ISSUE THREAD

**Issue by jenniferabowd** [npm/npm#17357]
#### I'm opening this issue because:

  - [ ] npm does not install new packages added to the package.json after the first npm install.

#### What's going wrong?
In the command line, when I run npm install after running it already (let's say a teammate adds a new package to the package.json), it does not install anything. I am using Node version 7.5.0 and NPM version 5.0.3.

#### How can the CLI team reproduce the problem?
Use Node version 7.5.0 (although that might not matter) and NPM version 5.0.3. Then clone a repo and hit npm install. Then add a package to the package.json. Then run npm install a second time. It will not download the new package.

**Comment by Cuel** [comment]
Did you add the package using npm install? Or directly in the package.json?
Sounds like the package is not in package-lock.json

**Comment by jenniferabowd** [comment]
Hello,

It happens when my teammate does npm install —save `package-name`. Then I pull down his changes and run npm install. The package does not download even though it is in the package.json. 

Best,

Jen

**Comment by Cuel** [comment]
Is there a package-lock.json or npm-shrinkwrap.json, and does it contain the package?

**Comment by jenniferabowd** [comment]
Hi,

I actually re-tested this with my teammate and now it is working. Sorry about that!

Best,

Jen

**Comment by jenniferabowd** [comment]
As an update, my teammate and I realized that the issue is we have NPM 5, but some of our teammates have NPM 4. When someone with NPM 4 downloads a package and saves it into the package.json, it does not have a package-lock.json. NPM 5 depends on the package-lock.json to download the dependencies. We have now notified all the developers at work and requested that they update their NPM to prevent this issue from happening again.
